modules/custom/az_event/az_event_trellis/config/install/az_event_trellis.settings.yml

```diff
@@ -1 +1 @@
-api_hostname: 'prod-trellis-gateway.trellis.arizona.edu'
+api_hostname: 'api.qa.trellis.arizona.edu'
```

Comment:
Is this what we want?

Reply:
TBD: to be reverted when the event search API is available on the production gateway.